data-prepper-api/src/main/java/com/amazon/dataprepper/model/event/DefaultEventMetadata.java

```diff
@@ -22,6 +22,8 @@
 
     private final String eventType;
 
+    private final String messageKey;
```

Comment:
#770 is suppose to enable the creation of an event object with a message key. 

Ideally we should be able to update our source plugins to create an event object without having to specify the key used. For example in the FileSource plugin we are using "message" as the key. See: [FileSource.java L90](https://github.com/opensearch-project/data-prepper/blob/main/data-prepper-plugins/common/src/main/java/com/amazon/dataprepper/plugins/source/file/FileSource.java#L90) & [107](https://github.com/opensearch-project/data-prepper/blob/main/data-prepper-plugins/common/src/main/java/com/amazon/dataprepper/plugins/source/file/FileSource.java#L107)

Ideally we should be able to create an event object storing "some value" with a **message** key: `Event event = JacksonEvent.builder().withMessageKey("some value")`. Resulting Event:
```
{
   "message": "some value"
}
```

Reply:
I missed @dlvenable review earlier. I agree with his suggestions.